addons/web/static/src/legacy/js/core/utils.js

```diff
@@ -1138,12 +1149,21 @@ var utils = Object.assign({
         return [
             '&',
             ['res_model', '=', 'ir.ui.view'],
-            '|',
-            '|',
-            ['name', '=like', '%.assets\_%.css'],
-            ['name', '=like', '%.assets\_%.js'],
-            ['name', '=like', '%.report_assets\_%.css'],
-        ];
+        ].concat(Array(assetsPatterns.length * 2 - 1).fill('|'))
+        .concat(...assetsPatterns.map(pattern => [
+            ['name', '=like', pattern],
+            ['name', '=like', pattern + '.map'],
+        ]));
+    },
+    /**
+     * Registers assets file name patterns.
+     *
+     * @param {Array} patterns compatible with the '=like' syntax
+     */
+    registerAssetsPatterns: function (patterns) {
+        for (const pattern of patterns) {
+            assetsPatterns.push(pattern);
+        }
```

Comment:
Dead code or am I missing something obvious?

Reply:
Maybe not obvious, but https://github.com/odoo/enterprise/pull/26681